prof/src/com/oracle/souffleprof/Tui.java

```diff
@@ -127,13 +127,8 @@ public void runProf() {
                 } else {
                     loadMenu();
                 }
-            }
-
-            if (!loaded) {
```

Comment:
I don't think it made any sense to check if it was successful here. It checks to make sure we've got a file loaded the top of the loop, which is the next thing hit after the if statement anyway, so it was redundant